### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by cryptal-mc**
**Main changes**
- Adapted miner and validators to using orchestrator-side learning rate scheduler rather than managing their own schedulers

**Behaviour, before and after**
Previously, each miner initialized its own learning rate scheduler object and used it to update the learning rate after each optimizer step. This is problematic for miner that join training later on since they will use a learning rate different from others.

In this new implementation, Only one learning rate scheduler object is defined on the orchestrator side, and incremented after the end of the merging phase. Miners simply query the API with a new endpoint called `/request_learning_rate` to get the current value of the learning rate.

### CODE REVIEW

**Review by Sarkosos — APPROVED** [review]
legit
